mautic/mautic
# remove obsolete scaffold files

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

a lot of stale files are still present in the `scaffold` folder + some files don't need to be scaffolded at all for composer based releases (e.g. the `Gruntfile.js`), as all development for mautic should happen via the monorepo, not the composer packages.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. run following command to trigger the console and npm actions
```
composer update --lock
```
3. run following commands to mimic a composer install, which needs the console file, npm files, ...
```
mkdir test_composer
cd test_composer
cp cp ../.github/ci-files/composer.json ./
composer install
```
